packages/ses/src/console.js

```diff
@@ -0,0 +1,102 @@
+import { globalThis, TypeError } from './commons.js';
```

Comment:
I’m inclined to rename this `reporting` rather than bundle it with console taming. I will probably continue to call them consoles, since a real console implements the needed interface. Sound good or no?

Reply:
(as we then later agreed verbally, the "them" we decided to call Reporters, where a Console is a structural subtype of Reporter)